app/routes/measure-concepts/new.js

```diff
@@ -11,4 +11,10 @@ export default class MeasureConceptsNewRoute extends Route {
       roadSignConcepts: this.store.findAll('road-sign-concept'),
     });
   }
+  resetController(controller) {
+    const measureConcept = controller.model.newMeasureConcept;
+    measureConcept.rollbackAttributes();
+    measureConcept.belongsTo('roadSignConcept').reload();
```

Comment:
Can you double check if this is needed? I thought new records would be deleted when you use `.rollbackAttributes()`. Can it even do a reload of a relationship that hasn't been saved yet?

Reply:
indeed, it wasn't needed, I removed them and will push in a second commit for clarity

Reply:
I'm not 100% sure but I think the rollbackAttributes on the new pages should have been kept. AFAIK now there will be "dangling" records in the store. (so if you enter the new route 5 times without saving, there will be 5 "new" records in the store.

Reply:
I checked it and it doesn't seem to dangle. can you join back the google meet so I can show you?